Backend/dev/migrations/rider-app/1125-add-merchantId-to-tables.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE atlas_app.booking_cancellation_reason ADD COLUMN merchant_id character(36) NOT NULL REFERENCES atlas_app.merchant (id) DEFAULT 'da4e23a5-3ce6-4c37-8b9b-41377c3c1a51';
```

Comment:
Just add merchantId as null in all tables without backfilling of data, as you have to make it Maybe in code. Also don't add any defaults.